test/geos_capi/multi_polygon_test.rb

```diff
@@ -37,4 +37,55 @@ def test_polygonize
 
     assert_equal expected, input.polygonize
   end
+
+  INVALID_MULTIPOLYGON = "MULTIPOLYGON (((0 1,2 1,2 2,0 2,0 1)), ((1 0,3 0,3 3,1 3,1 0)))"
+  LINEWORK_MULTIPOLYGON = "MULTIPOLYGON (((1 2, 1 1, 0 1, 0 2, 1 2)), " \
+                          "((1 3, 3 3, 3 0, 1 0, 1 1, 2 1, 2 2, 1 2, 1 3)))"
+  JOINED_POLYGON = "POLYGON ((0 2, 1 2, 1 3, 3 3, 3 0, 1 0, 1 1, 0 1, 0 2))"
```

Comment:
one small nitpick, but instead of defining these as constants can we make these private methods in the test class that return the wkt. A similar thing was done for a lot of the validity checks if you want something to reference. Keeping it as WKT is fine (I believe most of the validity tests return geometry objects), but this will make the style more consistent.

Reply:
same with the line_string test as well please

Reply:
that was done